DigitalLearningSolutions.Data.Migrations/2023072811401334_updateTermsAndConditions.cs

```diff
@@ -0,0 +1,176 @@
+﻿using FluentMigrator;
+using Microsoft.VisualBasic;
+
+namespace DigitalLearningSolutions.Data.Migrations
+{
+    [Migration(202307281140)]
+    public class UpdateTermsAndConditions : Migration
+    {
+        public override void Up()
+        {
+            var TermsAndConditions = @"<h1 class=""nhsuk-u-text-align-centre nhsuk-heading-xl"" >PLEASE   READ   THESE   TERMS   AND   CONDITIONS   CAREFULLY   BEFORE   USING   THE PLATFORM.  YOUR ATTENTION IS PARTICULARLY DRAWN TO THE PROVISIONS OF CLAUSE 14  (OUR  RESPONSIBILITY  FOR  LOSS  OR  DAMAGE  SUFFERED  BY  YOU)  AND  CLAUSE 15 (INDEMNITIES).</1>
```

Comment:
This will work but there are easier ways of doing this. I'd suggest storing the HTML to a file (see Binon's cookie policy html in Migrations/Resources for an example) and reading this into the migration to keep the migration class tidy and readable and the HTML easier to retrieve and edit.